insonmnia/hub/server.go

```diff
@@ -831,6 +840,13 @@ func New(ctx context.Context, cfg *HubConfig, version string) (*Hub, error) {
 		return nil, err
 	}
 
+	acl := NewACLStorage()
+	privateKey, err := x509.MarshalECPrivateKey(ethKey)
+	if err != nil {
+		return nil, err
+	}
+	acl.Insert(hex.EncodeToString(privateKey))
```

Comment:
For what do you add Hub's key in itself's Storage? 
There is ACLs for Workers, not for Hubs, isn't it?

Reply:
As we discussed this is default value to enable key sharing between hub and workers.